llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -72,10 +75,39 @@ static SmallVector<Value *> argVectorFlatten(CallInst *Orig,
   return NewOperands;
 }
 
+static VersionTuple getShaderModelVer(Module &M) {
+  std::string TTStr = M.getTargetTriple();
+  std::string Error;
+  auto Target = TargetRegistry::lookupTarget(TTStr, Error);
```

Comment:
We should already be in DirectX backend now.
Is it possible to have illegal TargetTriple?

Reply:
> We should already be in DirectX backend now. Is it possible to have illegal TargetTriple?

Deleted the redundant lookup and check in in-lined functionality.